OloEngine/tests/Lua/LuaBindingTest.cpp

```
@@ -0,0 +1,1987 @@
+#include "OloEngine/Gameplay/Inventory/InventoryComponents.h"
+#include "OloEngine/Gameplay/Quest/QuestComponents.h"
+#include "OloEngine/Gameplay/Abilities/AbilityComponents.h"
+#include "OloEngine/Scripting/Lua/LuaScriptGlue.h"
+
+#include "OloEngine/Scene/Scene.h"
+#include "OloEngine/Scene/Entity.h"
+#include "OloEngine/Scripting/C#/ScriptEngine.h"
+
+#include <glm/glm.hpp>
+
+#include <algorithm>
```

Cppcheck (2.20.0)

[information] Include file

(missingIncludeSystem)